snapcraft/parts/lifecycle.py

```diff
@@ -506,21 +506,35 @@ def _run_in_provider(
     if getattr(parsed_args, "enable_experimental_ua_services", False):
         cmd.append("--enable-experimental-ua-services")
 
+    project_path = Path().absolute()
     output_dir = utils.get_managed_environment_project_path()
 
     emit.progress("Launching instance...")
     with provider.launched_environment(
         project_name=project.name,
-        project_path=Path().absolute(),
+        project_path=project_path,
         base=project.get_effective_base(),
-        bind_ssh=parsed_args.bind_ssh,
         build_on=project.get_build_on(),
         build_for=project.get_build_for(),
         http_proxy=parsed_args.http_proxy,
         https_proxy=parsed_args.https_proxy,
     ) as instance:
         try:
             with emit.pause():
+                # mount project
```

Comment:
Why inside the paused emitter? Do these `mount` calls pollute the screen somehow?

Reply:
Good point.  I moved them outside of the `pause` and `try` calls.